### ISSUE TITLE
cryptography 35.0.0 breaks dns-lexicon (and certbot)

### ISSUE THREAD

**Issue by Schamschula** [AnalogJ/lexicon#969]
MacPorts updated `py-cryptography` to version 35.0.0 a couple of days ago. Now `certbot` is broken for those using `py-dns-lexicon`. When loading `certbot` we see the following error:

```
An unexpected error occurred:
pkg_resources.ContextualVersionConflict: (cryptography 35.0.0 (/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages), Requirement.parse('cryptography<4,>=2'), {'dns-lexicon'})

```
See: https://trac.macports.org/ticket/63567
and https://community.letsencrypt.org/t/cryptography-35-0-0-breaks-certbot/161813

**Comment by adferrand** [comment]
Yes, cryptography changed their version format. I am preparing a new release to relax the constraints and allow to use the latest version of cryptography.

**Comment by adferrand** [comment]
This is fixed with `3.8.0`.

**Comment by Schamschula** [comment]
Works for me!
Thanks!
